analytics/CAR-2021-01-009.yaml

```diff
@@ -0,0 +1,31 @@
+---
+title: Detecting Shadow Copy Deletion via vssadmin.exe
+submission_date: 2020/12/11
+information_domain: 'Host'
+platforms:
+  - Windows
+subtypes:
+  - Process
+analytic_types:
+  - TTP
+contributors:
+  - Cyware Labs
+id: CAR-2021-01-009
+description: |-
+  After compromising a network of systems, threat actors often try to delete Shadow Copy in an attempt to prevent administrators from restoring the systems to versions present before the attack. This is often done via vssadmin, a legitimate Windows tool to interact with shadow copies. This non-detection of this technique, which is often employed by ransomware strains such as “Olympic Destroyer”, may lead to a failure in recovering systems after an attack.
+  - technique: T1490
+    coverage: Low
+    subtechniques:
+      - None
+    tactics:
+      - TA0040
+implementations:
+- name: Splunk query for Detecting Shadow Copy Deletion via vssadmin.exe
+
+  description:  This query looks for the specific use of vssadmin in correlation to a deleting function, alerting us of attempts to delete shadow copies that possibly indicate malicious activity.
+  code: |-
+    index = *  CommandLine:"delete" OriginalFileName:"VSSADMIN.EXE" | table _time host User CommandLine
```

Comment:
This doesn't look like it was every tested, as I don't think it would work as is. Make sure you're committing tested analytics. Also leave out the table/results formatting. 
Should be something like: index = wineventlogs EventType=4688  CommandLine:"*delete*" OriginalFileName:"C:\\Windows\\System\\VSSADMIN.EXE"